lms/static/styles/components/_URLPicker.scss

```diff
@@ -0,0 +1,39 @@
+@keyframes validationErrorOpen--url {
```

Comment:
I had to override some of the css to get the message to fit better on the URLPicker.
300px vs 200px on the width.

Things are still a bit hard coded due to absolute pos, perhaps there is opportunity to improve, and generalize, but not at this juncture.